### DISCUSSION TITLE
Management UI for Crossplane

### DISCUSSION THREAD

**Discussion by illegalnumbers** [crossplane/crossplane#4794] [Knowledge base]
Is there a management UI for crossplane like akin to argo-workflow or argocd or kiali?

**Comment by MoeidHeidari** [comment]
here is an open-source project recently published. I guess there are a lot of people waiting for a new Crossplane uI out there. 

https://github.com/corpobit/crossview

you can install it easily using helm also. https://artifacthub.io/packages/helm/crossview/crossview.

feel free to contribute also should you have any idea that can make it even better.
